src/main/java/org/broadinstitute/hellbender/engine/GATKTool.java

```diff
@@ -268,6 +279,8 @@ protected void onStartup() {
         validateSequenceDictionaries();
 
         checkToolRequirements();
+
+        progressMeter = new ProgressMeter(recordsPerProgressUpdate);
```

Comment:
I think this should be extracted out to a method that subclasses can override. Maybe `initializeProgressMeter` or something. That way, subclasses can have a say in what the default rate of updates is (and not delegate all to commandline options). For example perhaps the default rate for variant walkers should be smaller than read walkers etc.